package.json

```diff
@@ -129,7 +130,8 @@
       "^three/.*$": "identity-obj-proxy",
       "^@babylonjs.*$": "identity-obj-proxy",
       "^.*/modules/curations/collectionCuration/toasts.*$": "identity-obj-proxy",
-      "@dcl/ecs": "identity-obj-proxy"
+      "@dcl/ecs": "identity-obj-proxy",
+      "@dcl/single-sign-on-client": "identity-obj-proxy"
```

Comment:
As the module is an ESM and not CommonJs, jest doesn't like it. Doing this will replace it in tests with a mockable stub.